src/main/kotlin/no/nav/familie/ef/iverksett/domene/Iverksett.kt

```diff
@@ -1,31 +1,52 @@
 package no.nav.familie.ef.iverksett.domene
 
+import no.nav.familie.kontrakter.ef.felles.StønadType
 import java.time.LocalDate
 import java.util.*
 
 data class Iverksett(
-    val forrigeTilkjentYtelse: TilkjentYtelse? = null,
-    val tilkjentYtelse: TilkjentYtelseMedMetaData,
-    val inntekt: List<Inntekt> = emptyList(),
-    val fagsakId: String,
-    val saksnummer: String? = null,
-    val behandlingId: String,
-    val eksternId: Long,
-    val relatertBehandlingId: String? = null,
-    val kode6eller7: Boolean,
-    val tidspunktVedtak: LocalDate? = null,
-    val vilkårsvurderinger: List<Vilkårsvurdering> = emptyList(),
-    val personIdent: String,
-    val barn: List<Person> = ArrayList(),
-    val behandlingType: BehandlingType,
-    val behandlingÅrsak: BehandlingÅrsak,
-    val behandlingResultat: BehandlingResultat,
-    val vedtak: Vedtak? = null,
-    val opphørÅrsak: OpphørÅrsak,
-    val aktivitetskrav: Aktivitetskrav,
-    val funksjonellId: String
+        val fagsak: Fagsakdetaljer,
+        val behandling: Behandlingsdetaljer,
+        val søker: Søker,
+        val vedtak: Vedtaksdetaljer,
 )
 
+data class Fagsakdetaljer(
+        val fagsakId: UUID,
+        val eksternId: Long,
+        val stønadstype: StønadType
+)
+
+data class Søker(
+        val aktivitetskrav: Aktivitetskrav,
+        val personIdent: String,
+        val barn: List<Barn> = ArrayList(),
+        val tilhørendeEnhet: String,
+        val kode6eller7: Boolean,
+)
+
+data class Vedtaksdetaljer(
+        val vedtak: Vedtak,
+        val vedtaksdato: LocalDate,
+        val opphørÅrsak: OpphørÅrsak?,
+        val saksbehandlerId: String,
+        val beslutterId: String,
+        val tilkjentYtelse: TilkjentYtelse,
+        val inntekter: List<Inntekt>
+)
+
+data class Behandlingsdetaljer(
+        val forrigeBehandlingId: String? = null,
+        val behandlingId: UUID,
+        val eksternId: Long,
```

Comment:
Burde det holde med bare `eksternId`, som kanskje burde hete `behandlingId` (men altså være Long)?

Reply:
Tenker nesten det kan være fint å ha begge, slik att vi kan logge behandlingId rundt om :) Enklere for oss å følge loggen hvis vi kan forholde oss til en type "behandlingId" internt i våret system :)